libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -212,6 +213,25 @@ export class ModelOverview extends React.Component<
       IsMulticlass(this.context.jointDataset.getModelType())
     );
 
+    const selectableAggregateMethods : IComboBoxOption[] = [
+      { key: localization.ModelAssessment.ModelOverview.metricTypes.macro,
+        text: localization.ModelAssessment.ModelOverview.metricTypes.macro },
+      { key: localization.ModelAssessment.ModelOverview.metricTypes.micro,
+        text: localization.ModelAssessment.ModelOverview.metricTypes.micro }
+    ];
+
+    const selectableClassNames : IComboBoxOption[] = []
+    if (this.context.dataset.class_names) {
+      for (var class_name of this.context.dataset.class_names) {
```

Comment:
please resolve the linting issues - note in typescript we use camelCase instead of the underscore like in python

Reply:
also use let instead of var here